lean4-infoview/src/infoview/info.tsx

```diff
@@ -180,7 +190,9 @@ function GoalInfoDisplay(props: GoalInfoDisplayProps) {
     return (
         <>
             <LocationsContext.Provider value={locs}>
-                <FilteredGoals key="goals" headerChildren="Tactic state" initiallyOpen goals={goals} displayCount />
+                <span data-vscode-context={JSON.stringify({ selectedLocationsId })}>
```

Comment:
Should this be part of `FilteredGoals` (and also be dependent on the presence of `LocationsContext`)?